### ISSUE TITLE
annotations of EGTEA Gaze+

### ISSUE THREAD

**Issue by yihuang995** [fpv-iplab/rulstm#13]
Dear author, thanks for your work and code release!

I am working on EGTEA Gaze+ dataset now. I do not clearly know how you process the raw annotations of this dataset to fit this project.

Could you please provide the annotation files which are compatible to this project? Such as action.csv, many_shot_nouns.csv, many_shot_vebs.csv and training.csv.

Thank you again!

**Comment by antoninofurnari** [comment]
Hello,

We are not officially supporting training and testing on EGTEA Gaze+. However, I have now added to the repository the csvs which I used for the ICCV publication. You will find three training and three validation csv files. These are related to the three EGTEA Gaze+ splits. Also actions.csv is included, while many_shot_nouns.csv and many_shot_verbs.csv are not included because this dataset does not define lists of many shot labels. 

Hope the files will be helpful.

Best,
Antonino
